client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -109,14 +110,23 @@ export const SitesGridItem = memo( ( props: SitesGridItemProps ) => {
 	const isWpcomStagingSite = isStagingSite( site );
 	const translatedStatus = useSiteLaunchStatusLabel( site );
 	const isTrialSitePlan = useSelector( ( state ) => isTrialSite( state, site.ID ) );
+	const isWpAdminDefaultSite = useSelector( ( state ) =>
+		getSiteOption( state, site.ID, 'wpcom_admin_interface' )
+	);
+	const hasHostingFeatures = ! isNotAtomicJetpack( site ) && ! isP2Site;
```

Comment:
Is there a way we can write this so it checks for the appropriate value, instead of asserting a double negative?

Reply:
@danielbachhuber t[his line](https://github.com/Automattic/wp-calypso/blob/b20376f4ac3a5dd4d29f37a9f8394f92a83b320d/client/sites-dashboard/components/sites-grid-item.tsx#L113) already checks for the appropriate value e.g. whether the `wpcom_admin_interface` is set to `wp-admin`. Actually, only this check is already enough to make the links redirect to wp-admin because technically it is not possible to set this option on any other site but an Atomic site with the endpoint. I mentioned in the PR description that I added the check for the `hostingFeatures` to be extra sure because the option is still present on Jetpack and WPCOM Simple sites (even though it is empty) but it is not strictly necessary. 

As for the `hostingFeatures`, I am not sure how you would check for the appropriate value just for the hosting features? I looked for similar implementations and [found this as an example ](https://github.com/Automattic/wp-calypso/blob/b20376f4ac3a5dd4d29f37a9f8394f92a83b320d/client/sites-dashboard/components/sites-ellipsis-menu.tsx#L399) so that's what I was following. 

Happy to remove this check if you don't think it is necessary.

Reply:
@katinthehatsite I think the check is necessary. This condition is ugly, though:

```
! isNotAtomicJetpack( site ) && ! isP2Site
```

Can we avoid the double negative?

Reply:
I made some changes in f5071d316c5b6944d5154cba3b36bdfc0329c8d1 to avoid double-negative.